dbt/models/default/schema/default.vw_pin_value.yml

```diff
@@ -118,28 +118,31 @@ models:
           - `3` = Board certified
       - name: year
         description: '{{ doc("shared_column_year") }}'
+        data_tests:
+          - not_null:
+              name: default_vw_pin_value_stage_num_not_null
```

Comment:
Added this test in while I was here, since it was quick and easy and the equivalent `stage_name` test ended up being helpful in catching some problems.